pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -250,20 +264,35 @@ func (pl *InterPodAffinity) PreFilter(ctx context.Context, cycleState *framework
 	if podInfo.ParseError != nil {
 		return framework.NewStatus(framework.UnschedulableAndUnresolvable, fmt.Sprintf("parsing pod: %+v", podInfo.ParseError))
 	}
+	podNS, err := pl.nsLister.Get(pod.Namespace)
+	if err != nil {
+		return framework.NewStatus(framework.Error, fmt.Sprintf("failed to get pod namespace object: %v", err))
```

Comment:
what about just assuming empty labels so that tests don't need to create fake namespaces?

In production it shouldn't happen, as the pod would get deleted.... unless we want to error in the case where the pod was not garbage collected yet?

Reply:
tests are already handled, I rather not to ignore the error.

Reply:
ah, the integration tests needs updating...

Reply:
ok, so I changed it such that we assume empty labels. It is actually surprising to me that integration and e2e tests failed since we stand up an actual api-server, but perhaps not with the configuration that verifies the existence of the namespace on pod creation.